lib/Nitrapi/Customer/Registration.php

```diff
@@ -15,6 +15,9 @@ public function __construct(Nitrapi $api,
                                 $userName,
                                 $email,
                                 $password,
+                                $consentPrivacy,
```

Comment:
Maybe default to false?

Reply:
If it's false, the NitrAPI will throw an exception anyway. Without a default, library users can see the broken backwards-compatibility in a more "static" way.